client/src/Components/Common/DateTimePicker/index.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import { TextField } from '@mui/material';
+import moment from 'moment';
+
+export default function DateTimePicker() {
+  const [dateTime, setDateTime] = useState(moment().format('yyyy-MM-DDThh:mm'));
+
+  const onChangeHnadler = (e) => {
+    setDateTime(e.target.value);
+  };
```

Comment:
we'll control state from outer component som dateTime and changeHandler should be a props for the component,
this component should be representational